source/MRMesh/MRBitSetParallelFor.h

```diff
@@ -12,27 +13,49 @@ namespace MR
 /// executes given function f for each bit in bs in parallel threads;
 /// it is guaranteed that every individual block in bit-set is processed by one thread only
 template <typename BS, typename F>
-void BitSetParallelForAll( const BS & bs, F f )
+void BitSetParallelForAll( const BS& bs, F f, SimpleProgressCallback progressCb = {} )
```

Comment:
It looks like, two BitSetParallelForAll functions will be better: one old, and the other with progress callback and static partitioning